src/menu/views/settings_editor.c

```diff
@@ -9,11 +12,104 @@ static const char *format_switch (bool state) {
     }
 }
 
+static void set_pal60_type (menu_t *menu, void *arg) {
+    bool initial_setting_val = menu->settings.pal60_enabled;
+    menu->settings.pal60_enabled = (bool) (arg);
+    if (initial_setting_val != menu->settings.pal60_enabled) {
+        settings_changed = true;
+    }
+    
+    menu->browser.reload = true;
```

Comment:
this is not necessary here at all, it causes current folder open in the browser to be reloaded so doing that here is pointless